meson/post_install.py

```diff
@@ -3,7 +3,9 @@
 import os
 import subprocess
 
-schemadir = os.path.join(os.environ['MESON_INSTALL_PREFIX'], 'share', 'glib-2.0', 'schemas')
+prefix = os.environ.get('MESON_INSTALL_PREFIX')
+datadir = os.path.join(prefix, 'share')
```

Comment:
This should be `sys.argv[1]` and `join_paths(get_option('prefix'), get_option('datadir'))` should be passed as an argument to `add_install_script`:

https://github.com/lainsce/quilter/blob/214770094f40735b7d1c0ac08ef3e783ff888810/meson.build#L88

Reply:
gotcha